egs/babel/s5d/local/run_asr_segmentation.sh

```diff
@@ -0,0 +1,336 @@
+#! /bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+# Features configs
+feat_type=plp
+feat_config=conf/plp.conf
+add_pitch=true
```

Comment:
These are related to the GMM models used for alignment and decoding. Pitch is used by default in Babel recipes.